DatadogRUM/Sources/RUMMonitorProtocol.swift

```diff
@@ -53,6 +53,14 @@ public protocol RUMMonitorProtocol: AnyObject {
 
     // MARK: - session
 
+    /// Get the currently active session ID. Returns `nil` if no sessions are currently active or if
+    /// the current session is sampled out.
+    /// This method uses an asynchronous callback to ensure all pending RUM events have been processed
+    /// up to the moment of the call.
+    /// - Parameters:
+    ///   - completion: the callback that will recieve the current session ID.
+    func getCurrentSessionID(completion: @escaping (String?) -> Void)
```

Comment:
suggestion/ The `get*` prefix isn't common in Swift, so `currentSessionID(completion:)` might be worth considering. Also, as we're dispatching a callback here, it is worth adding a note on what thread it will be called on (notably: not the main thread in current implementation).